packages/create-project/bin/create-project.js

```diff
@@ -22,5 +23,6 @@ const name = basename(directory);
 env.run('create-project', {
   directory,
   name,
+  registry: cli.registry,
```

Comment:
Is this change so that the global yarn registry setting doesn't need to be adjusted? (And so can be removed from the script.) If so, this would be good since means running these tests don't affect global state locally.

Reply:
In theory, yes. You can either pass `--registry` during `create-project`, or you can specify the registry in the test generator (see the `project()` function in `test/cli_test`).